package.json

```diff
@@ -13,12 +13,12 @@
     "commander": "^2.9.0",
     "handlebars": "^4.0.6",
     "isomorphic-fetch": "^2.2.1",
-    "progress": "https://github.com/visionmedia/node-progress",
+    "progress": "^2.0.0",
     "ramda": "^0.23.0",
     "string": "^3.3.3",
     "to-pascal-case": "^1.0.0"
   },
   "devDependencies": {
-    "@procore/js-sdk-endpoints": "^1.8.1"
+    "@procore/js-sdk-endpoints": "^1.9.1"
```

Comment:
requires itself as a dev dependency?

Reply:
Yep. That shouldn't be here. Thanks for catching.